mongo_connector/doc_managers/doc_manager_base.py

```diff
@@ -20,6 +20,10 @@
 class DocManagerBase(object):
     """Base class for all DocManager implementations."""
 
+    version_info = tuple()
+    version = 'unkown'
```

Comment:
Changed to be defined at the DocManager implementation level. We can't use the `__init__.py` files as explained above. DocManager versions should probably be put on hold until we figure out what to do about the bigger problem.